client-core/src/service/wallet_service.rs

```diff
@@ -248,28 +290,73 @@ where
     }
 
     /// Get the wallet state from storage
+    // storage -> wallet
     pub fn get_wallet_state(&self, name: &str, enckey: &SecKey) -> Result<WalletState> {
         load_wallet_state(&self.storage, name, enckey)?.err_kind(ErrorKind::InvalidInput, || {
             format!("WalletState with name ({}) not found", name)
         })
     }
 
+    /// Save wallet to storage
+    pub fn save_wallet(&self, name: &str, enckey: &SecKey, wallet: &Wallet) -> Result<()> {
+        self.storage
+            .save_secure(KEYSPACE, name, enckey, wallet)
+            .expect("save walet-name in save_wallet");
```

Comment:
There is a lot of places where `expect` is used. Proper error handling will be better.

Reply:
good point, i'll fix it